index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <header>
+    <h1>My logo</h1>
+    <div id="hamburgerContainer">
+      <div class="hamburgerBar">&nbsp;</div>
+      <div class="hamburgerBar">&nbsp;</div>
+      <div class="hamburgerBar">&nbsp;</div>
```

Comment:
- [ ] Kindly adjust the semantics of the HTML here on the hamburger to have s a `nav` tag.

- The `nav` tag is for semantics reasons, and the right position for convention and user experience for mobile devices.